.github/workflows/cicd.yaml

```diff
@@ -120,4 +120,42 @@ jobs:
         with:
            name: trivy-report
            path: trivy-report.json
-  
\ No newline at end of file
+
+  integration-test:
+    runs-on: ubuntu-latest
+    needs: [bandit, safety, trivy]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install yq and bash
+        run: |
+          sudo snap install yq
+          sudo apt update
+          sudo apt install bash -y
+
+      - name: Build Connaisseur image
+        run: |
+          sed -i "s+TAG =.*+TAG = ${RELEASE_VERSION}+" Makefile
+          sed -i "s+IMAGE_NAME =.*+IMAGE_NAME = ${RELEASE_IMAGE}+" Makefile
+          make docker
+
+      - name: Creating KinD cluster
+        run: |
+          GO111MODULE="on" go get sigs.k8s.io/kind
+          kind create cluster --wait 120s
+
+      - name: Check KinD cluster
+        run: |
+          kubectl get nodes
+
+      - name: Add images to KinD
+        run: |
+          kind load docker-image ${RELEASE_IMAGE}:${RELEASE_VERSION}
+          kind load docker-image ${RELEASE_IMAGE}:helm-hook
+
+      - name: Run actual integration test
+        env:
+          NOTARY_USER: ${{ secrets.DOCKER_USERNAME }}
+          NOTARY_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
these are not used/needed? Probably, there due to the originally required auth to notary...

Reply:
Correct, will remove

Reply:
probably should also delete the creds from github...for now...

Reply:
or keep if you assume the next step is up soon...

Reply:
Would keep since I guess I'm doing release soonish